build/commands/lib/util.js

```diff
@@ -82,6 +82,46 @@ const util = {
     return prog
   },
 
+  /**
+   * Runs the given command line and pipes to stdout/stderr, except lines that match the given `header` and `body`
+   * regular expressions. This can be used to suppress warnings of the form:
+   *   In file X:
+   *     Superfluous warning 1
+   *     Superfluous warning 2
+   *     ...
+   * Here, "In file X" is the header and "Superfluous warning" are body lines. Body lines must immediately follow the
+   * header and headers that are not followed by a body line are not silenced.
+   */
+  runFiltered: (cmd, args=[], options = {}, header=/.^/, body=/.^/) => {
```

Comment:
does this still output with correct tty formatting so you don't get a long list of every target in the command line locally?